### PR TITLE
[PyKDL] Extended constructor LMA solver

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Review and merge after: https://github.com/orocos/orocos_kinematics_dynamics/pull/294

Fixes https://github.com/orocos/orocos_kinematics_dynamics/issues/292

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
@meyerj ping ;)

**Review by meyerj — APPROVED** [review]
LGTM!

File: python_orocos_kdl/PyKDL/kinfam.cpp [hunk 1]
```diff
@@ -436,6 +436,9 @@ void init_kinfam(pybind11::module &m)
     // ChainIkSolverPos_LMA
     // -------------------------
     py::class_<ChainIkSolverPos_LMA, ChainIkSolverPos> chain_ik_solver_pos_LMA(m, "ChainIkSolverPos_LMA");
+    chain_ik_solver_pos_LMA.def(py::init<const Chain&, const Eigen::Matrix<double,6,1>&, double, int, double>(),
```
Comment (meyerj) [inline python_orocos_kdl/PyKDL/kinfam.cpp:439]: Good to know that pybind11 can deal with Eigen types directly:
https://pybind11.readthedocs.io/en/stable/advanced/cast/eigen.html

Do you need to
```cpp
#include <pybind11/eigen.h>
```
?

Comment (meyerj) [reply python_orocos_kdl/PyKDL/kinfam.cpp:439]: Ah, already done here:
https://github.com/orocos/orocos_kinematics_dynamics/blob/ae9cdbb252a150781a362c1b34f54953a2004aae/python_orocos_kdl/PyKDL/PyKDL.h#L26

**Pull request merged**